src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.scss

```diff
@@ -21,6 +24,7 @@
   &__room-title {
     position: absolute;
     left: 50%;
+    bottom: -25px;
```

Comment:
Why -25? Can reducing the bottom padding help you with achieve the same result?

Reply:
I set 0 but it is inside the image so that's I set negative to put it down.
I could not find the bottom padding?

Reply:
https://github.com/sparkletown/sparkle/pull/1976/files#diff-fa48b4129d3054c897d06038e93337f52f088e7e67f9a501ff5f4b7f2d1405e5R34

This is the padding, it's currently set, horizontal and vertical. You can reduce the bottom by adding the other 2 values.